packages/theming/theme-tokens/src/getOfficeAliasTokens.ts

```diff
@@ -0,0 +1,18 @@
+import * as colorfulAliasTokens from './colorful/tokens-alias';
+import * as darkGrayAliasTokens from './darkGray/tokens-alias';
+import * as blackAliasTokens from './black/tokens-alias';
+import * as hcAliasTokens from './highContrast/tokens-alias';
+
+export function getOfficeAliasTokens(officeTheme: string) {
+  if (officeTheme === 'White' || officeTheme === 'Colorful') {
+    return colorfulAliasTokens.default;
```

Comment:
White and Colorful theme use the same colors/palette?

Reply:
Yep, they're exactly the same right now. That may change in the future, but that should just mean we would add a new input file for White theme, or something